docs/design/disable-add-in-commands.md

```diff
@@ -16,11 +16,18 @@ You can also specify whether the command is enabled or disabled when the Office
 >
 > [Basic concepts for Add-in Commands](add-in-commands.md)
 
+## Preview status
+
+The APIs described in this article are in preview and are currently only available in Excel.
+
+> [!NOTE]
+> [!INCLUDE [Information about using preview APIs](../includes/using-preview-apis.md)]
+
 ## Rules and gotchas
 
 ### Single-line ribbon in Office on the web
 
-In Office on the web, the APIs and manifest markup described in this article only affect the single-line ribbon. They have no effect on the multiline ribbon. They affect both ribbons for desktop Office. For more information about the two ribbons, see [Use the simplified ribbon](https://support.office.com/article/Use-the-Simplified-Ribbon-44bef9c3-295d-4092-b7f0-f471fa629a98).
+In Office on the web, the APIs and manifest markup described in this article only affect the single-line ribbon. They have no effect on the multiline ribbon. They affect both ribbons for desktop Office. For more information about the two ribbons, see [The new look of Office - Simplified Ribbon](https://support.microsoft.com/office/the-new-look-of-office-a6cdf19a-b2bd-4be1-9515-d74a37aa59bf).
```

Comment:
Remember to use the guidance in our OneNote for how to make the support.microsoft.com locale-agnostic. So in this case, the link should likely be updated to something like the following.
```suggestion
In Office on the web, the APIs and manifest markup described in this article only affect the single-line ribbon. They have no effect on the multiline ribbon. They affect both ribbons for desktop Office. For more information about the two ribbons, see [The new look of Office - Simplified Ribbon](https://support.microsoft.com/office/a6cdf19a-b2bd-4be1-9515-d74a37aa59bf).
```

Reply:
Thanks for the email heads up on this, I must have missed this note to do this! :)